packages/extension-api/src/extension-api.d.ts

```diff
@@ -3510,7 +3510,7 @@ declare module '@podman-desktop/api' {
     /**
      * Attempt to pull the image even if an older image exists locally.
      */
-    pull?: string;
+    pull?: boolean;
```

Comment:
for now needs to be `string | boolean` there

and later if a string we convert it to true or false

Reply:
I updated the extension loader to properly convert potential string to boolean